scripts/zones/Lower_Jeuno/Zone.lua

```diff
@@ -81,16 +84,42 @@ end;
 
 function onGameHour()
     local VanadielHour = VanadielHour();
-
+    local playerOnQuestId = GetServerVariable("[JEUNO]CommService");
```

Comment:
o ya
derp

disregard my other comments though i'd prefer the players in zone notified to light lamps before merging